docs/source/install.md

```diff
@@ -252,64 +270,92 @@ until [ -z "$(docker service ls --filter label=com.docker.stack.namespace=secret
 echo "flowapi" | docker secret create FLOWAPI_FLOWDB_USER -
-openssl rand -base64 16 | tr -cd '0-9-a-z-A-Z' | docker secret create FLOWAPI_FLOWDB_PASSWORD -
-openssl rand -base64 16 | tr -cd '0-9-a-z-A-Z' | docker secret create POSTGRES_PASSWORD -
 openssl rand -base64 16 | tr -cd '0-9-a-z-A-Z' | docker secret create REDIS_PASSWORD -
+
+# FlowDB
+openssl rand -base64 16 | tr -cd '0-9-a-z-A-Z' | docker secret create FLOWAPI_FLOWDB_PASSWORD -
+openssl rand -base64 16 | tr -cd '0-9-a-z-A-Z' | docker secret create FLOWDB_POSTGRES_PASSWORD -
+
+# FlowAPI
+openssl rsa -pubout -in  tokens-private-key.key -out tokens-public-key.pub
+docker secret create PRIVATE_JWT_SIGNING_KEY tokens-private-key.key
+docker secret create PUBLIC_JWT_SIGNING_KEY tokens-public-key.pub
 echo "flowapi_server" | docker secret create FLOWAPI_IDENTIFIER -
 openssl req -newkey rsa:4096 -days 3650 -nodes -x509 -subj "/CN=flow.api" \
     -extensions SAN \
     -config <( \
-        cat $(   ( [[ -e /System/Library/OpenSSL/openssl.cnf ]] \
-                   && echo "/System/Library/OpenSSL/openssl.cnf" ) \
-              || ( [[ -e /etc/ssl/openssl.cnf ]] \
-                   && echo "/etc/ssl/openssl.cnf" ) \
-              || ( [[ -e /etc/pki/tls/openssl.cnf ]] \
-                   && echo "/etc/pki/tls/openssl.cnf" ) ) \
+        cat $(   ( [[ -e /System/Library/OpenSSL/openssl.cnf ]] && echo "/System/Library/OpenSSL/openssl.cnf" ) \
+              || ( [[ -e /etc/ssl/openssl.cnf ]] && echo "/etc/ssl/openssl.cnf" ) \
+              || ( [[ -e /etc/pki/tls/openssl.cnf ]] && echo "/etc/pki/tls/openssl.cnf" ) ) \
     <(printf "[SAN]\nsubjectAltName='DNS.1:localhost,DNS.2:flow.api'")) \
     -keyout cert.key -out cert.pem
-# NOTE: make sure that the previous command generated _both_ files cert.key and cert.pem
+if ! ( [ -e cert.key ] && [ -e cert.pem ] ); then
+    echo "Generation of the SSL certificate failed."
+    echo "Please the check the command (in particular the path to the openssl.cnf file) and try again."
+    exit
+fi
 cat cert.key cert.pem > cert-flowkit.pem
 docker secret create cert-flowkit.pem cert-flowkit.pem
-echo "secret" | docker secret create JWT_SECRET_KEY -
+
+# Deploy the stack
 
 echo "Deploying stack"
-docker stack deploy --with-registry-auth -c docker-stack.yml secrets_test
+docker stack deploy -docker-stack.yml secrets_test
 ```
 
-This will bring up a single node swarm, create random 16 character passwords for the database users, generate a certificate valid for the `flowkit.api` domain (and point that to `localhost` using `/etc/hosts`), pull all necessary containers, and bring up the API with `secret` as the JWT secret key.
+This will bring up a single node swarm, create random 16 character passwords for the database users, generate a certificate valid for the `flowkit.api` domain (and point that to `localhost` using `/etc/hosts`), pull all necessary containers, and bring up the API with a newly generated JWT signing key pair.
```

Comment:
Isn't the statement "This will [...] bring up the API with a newly generated JWT signing key pair" a bit confusing, because the key pair isn't generated in the API but rather in FlowAuth?